### ISSUE TITLE
[Feature] Snooze/Delay article

### ISSUE THREAD

**Issue by iyerusad** [FreshRSS/FreshRSS#4365]
**Is your feature request related to a problem? Please describe.**
Some feeds post content that flat out isn't available to view - e.g. Youtube will list a video that actually "Premieres in XX hours". It seems only one entry is added in their feed for this.

Additionally, there is sometimes stuff I'd like to read up on but don't have a easy way to "snooze for X time" - many Youtube movie reviews will be listed before chance to see a movie and would like to snooze/delay for like 2 weeks or 1 month (till get chance to watch it, and then compare notes).

**Describe the solution you’d like**

- A menu item to snooze current article, or perhaps button in the article view header.
  - Possible Snooze times: 1 hour, 3-6 hours, 1 day, 1 week, 1 month, 1 year, custom date till when
- Snoozed items would still be in the read tab, but perhaps with a snoozed emoji indicator
- A "Snoozed" category to review all currently snoozed videos

**Describe alternatives you’ve considered**

- I've used the star/favorite button and used that as a filter read later list.

**Additional context**
See this most with Youtube "premieres" videos, and perhaps would be neat if the Youtube plugin could detect that and auto-snooze on user's behalf - but imagine would need this kind of capability to be available.

**Comment by BlueWhale235** [comment]
About youtube, I have no ideas. Can you give me a rss url?  

**About second issue:** 
**I think that you can add a tag to the article which you want to see/mark**  . _But I tested, and I was unable to search by tag. It may a bug._

Now you can:  
![image](https://user-images.githubusercontent.com/35369979/168458059-3ade6562-d818-4e79-92a3-f1a15dc5ef71.png)  
Where to find articles by a tag:  
![image](https://user-images.githubusercontent.com/35369979/168458137-ae95cee0-46da-4b95-bbbd-ff62a809acc0.png)

**Comment by iyerusad** [comment]
> About youtube, I have no ideas. Can you give me a rss url?

https://www.youtube.com/feeds/videos.xml?channel_id=UCgUvk6jVaf-1uKOqG8XNcaQ

Most recent is video that "premieres" in ~30+ hours (at time of posting).

**Comment by BlueWhale235** [comment]
I checked briefly and did find this issue. I don't know much about youtube so it may take a special person to fix this. I guest that youtube is not adapted to rss.

**Comment by iyerusad** [comment]
> [...]to fix this.  youtube is not adapted to rss[...]

Not a bug (with FreshRSS or Youtube's RSS). New feature request/petition. Maybe gets added maybe no. I didn't see anything snooze-like feature in the issue board (closed or open).

**Comment by BlueWhale235** [comment]
@iyerusad  Do you want this ?:
![image](https://user-images.githubusercontent.com/35369979/168472518-4f75c557-3d8e-41ac-b644-ff232f2fbe03.png)
need extension: YouTube/PeerTube Video Feed  
But the video cannot play, may be youtube's limit.
the same as `theoldreader`
![屏幕截图 2022-05-15 201525](https://user-images.githubusercontent.com/35369979/168472493-4ebdabc2-c42b-4f4d-95ed-0400b0654d5b.png)

**Comment by iyerusad** [comment]
@BuleWhale No. I don't know how else to say this more plainly: You are not understanding the request. The video is not playing because it is currently blocked by the creator from playing until a certain date/time. That is okay - the feature I'd like is a way to snooze the article in FreshRSS to show up later, at some manually set time. Not a bug but rather a new feature request.

**Comment by BlueWhale235** [comment]
I'm not a native English speaker, I have times when I think of something else.. I am sorry for that.   
**So add a timer to remind you to do something?** I think this is an extension feature.

**Comment by iyerusad** [comment]
> I'm not a native English speaker, I have times when I think of something else.. I am sorry for that. **So add a timer to remind you to do something?** I think this is an extension feature.

That's totally okay. Yes this is an extension feature.
